kernel/fstdata.cc

```diff
@@ -29,7 +29,6 @@ static std::string file_base_name(std::string const & path)
 
 FstData::FstData(std::string filename) : ctx(nullptr)
 {
-	#if !defined(YOSYS_DISABLE_SPAWN)
```

Comment:
The behaviour changes here, but I think there would be an error later when trying to open the fst file

Reply:
Yes, this is probably fine to remove.